charts/pact-broker/templates/deployment.yaml

```diff
@@ -93,10 +93,18 @@ spec:
             - name: PACT_BROKER_DATABASE_USERNAME
               value: {{ include "broker.databaseUser" . }}
             - name: PACT_BROKER_DATABASE_PASSWORD
+              {{- if or .Values.postgresql.auth.existingSecret .Values.externalDatabase.config.auth.existingSecret }}
               valueFrom:
                 secretKeyRef:
                   name: {{ include "broker.databaseSecretName" . }}
                   key: {{ include "broker.databaseSecretKey" . }}
+              {{- else }}
+                {{- if and .Values.postgresql.enabled .Values.postgresql.auth.password }}
+              value: {{ .Values.postgresql.auth.password }}
+                {{- else if and .Values.externalDatabase.enabled .Values.externalDatabase.config.auth.password }}
+              value: {{ .Values.externalDatabase.config.auth.password }}
```

Comment:
This needs to be `value: {{ .Values.externalDatabase.config.auth.password | quote }}`

Reply:
```suggestion
              value: {{ .Values.externalDatabase.config.auth.password | quote }}
```